### DISCUSSION TITLE
Bug: New line doesn't work with "stage.random.text"

### DISCUSSION THREAD

**Discussion by MUGEN-Slayer** [ikemen-engine/Ikemen-GO#933] [Feature Request]
It's like the title describes. I'm using it like this:

`stage.random.text = "Battle Location: \n???"`

The result on the select screen when random stage is highlighted:
`Battle Location: \n???`

With the named stages, the new line works:
`stage.text = "Battle Location: \n%s"`

**Platform**
 - 0.98.2
 - Windows 10, AMD Radeon RX Vega 10 (onboard).

**Comment by Lazin3ss** [comment]
We should add newline support in most, if not all text definitions. After k4thos' lua refactoring effort, this will be doable.
